### PR TITLE
[new locale] Added locale ug-cn

### PR DESCRIPTION
**Pull request by boyaq**
Hey, here is the locale and test file for uyghur language.

Thanks.

### CODE REVIEW

**Comment by marwahaha** [comment]
@boyaq - one more change. Can you add to `component.json`? Similar to this PR
https://github.com/moment/moment/commit/230bfc1afd0bb9481f2b0506cb991cda5c49de16

File: src/locale/ug-cn.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+//! locale : Uyghur (China) [ug-CN]
+//! author: boyaq : https://github.com/boyaq
+
+import moment from '../moment';
+
+export default moment.defineLocale('ug-cn', {
+    months: 'يانۋار_فېۋرال_مارت_ئاپرېل_ماي_ئىيون_ئىيول_ئاۋغۇست_سىنتەبىر_ئۆكتەبىر_نويابىر_دىكابىر'.split(
+        '_'
+    ),
+    monthsShort: 'يانۋار_فېۋرال_مارت_ئاپرېل_ماي_ئىيون_ئىيول_ئاۋغۇست_سىنتەبىر_ئۆكتەبىر_نويابىر_دىكابىر'.split(
```
Comment (JLHwung) [inline src/locale/ug-cn.js]: I notice that there is some differences with the [Unicode CLDR Data](http://www.unicode.org/repos/cldr/tags/release-32-0-1/common/main/ug.xml) (see calendar type="gregorian" => months => monthsContext type="format" => monthWidth type="abbreviated" => month). 
The Unicode CLDR is
```
<monthWidth type="abbreviated">
<month type="1">يانۋار</month>
<month type="2">فېۋرال</month>
<month type="3">مارت</month>
<month type="4">ئاپرېل</month>
<month type="5">ماي</month>
<month type="6">ئىيۇن</month>
<month type="7">ئىيۇل</month>
<month type="8">ئاۋغۇست</month>
<month type="9">سېنتەبىر</month>
<month type="10">ئۆكتەبىر</month>
<month type="11">نويابىر</month>
<month type="12">دېكابىر</month>
</monthWidth>
```
Are they both acceptable in `ug-cn`?

Comment (boyaq) [reply src/locale/ug-cn.js]: You are right, it's a typo. And thanks for the link, I changed the translation and made it compatible with unicode cldr data.

**Comment by boyaq** [comment]
sorry guys have to make a new pull request. this contains old branch info.

**Pull request closed without merging**